extensions/chatgpt/.eslintrc.json

```diff
@@ -1,6 +1,16 @@
 {
   "root": true,
-  "extends": [
-    "@raycast"
-  ]
-}
\ No newline at end of file
+  "plugins": ["import"],
+  "extends": ["@raycast", "plugin:react-hooks/recommended"],
+  "rules": {
+    "import/order": [
+      "error",
+      {
+        "newlines-between": "always",
+        "alphabetize": {
+          "order": "asc"
+        }
+      }
+    ]
+  }
+}
```

Comment:
Why change this file back to an older version?

Reply:
oh. sry, I will revert this

Reply:
done